commands/lyrics.js

```diff
@@ -1,39 +1,164 @@
-import { SlashCommandBuilder } from 'discord.js'
+import { SlashCommandBuilder, EmbedBuilder, ButtonBuilder, ButtonStyle, ActionRowBuilder } from 'discord.js'
+import { userSongs } from '../commands/play.js'
+import Genius from 'genius-lyrics'
 
 export const command = new SlashCommandBuilder()
     .setName('lyrics')
     .setDescription('Asks Kermit to google for lyrics')
 
-command.aliases = ['l', 'lyric']
-
-command.slashRun = async function slashRun(client, interaction)
-{
-    await interaction.followUp({ content: 'Lets ask Kermit!', ephemeral: true });
+command.aliases = ['.l', '.lyric']
 
+command.slashRun = async function slashRun(client, interaction) {
     const channel = interaction.channel
     const send = channel.send.bind(channel)
+    const member = interaction.member
 
-    await run(client, channel, send)
+    await run(client, channel, send, member)
 }
 
-command.prefixRun = async function prefixRun(client, message, parameters)
-{
+command.prefixRun = async function prefixRun(client, message) {
     const channel = message.channel
+    const member = message.member
     const send = channel.send.bind(channel)
-    
-    await channel.send('Lets ask Kermit!')
 
-    await run(client, channel, send)
+    await run(client, channel, send, member)
 }
 
-async function run(client, channel, send)
-{
+async function run(client, channel, send, member) {
+
+    const lyricsToken = client.lyricsToken
+    const geniusClient = new Genius.Client(lyricsToken)
```

Comment:
It looks like client.lyricsToken is never set, maybe you forgot to commit that.
Maybe we can set client.genius = new Genius.Client instead of the token, that might be nicer.

Reply:
Added this change, it is now  client.genius rather than the constant geniusClient